### PR TITLE
Create template pull-request

### PR DESCRIPTION
**Pull request by Maciej818**

### CODE REVIEW

**Comment by Maciej818** [comment]
@AdrianoKF kind request for your review. I assumed that .github directory is suitable place for PR template.

**Comment by Maciej818** [comment]
@AdrianoKF True, there is only .md option for PR (it was my wishful thinking to do this in yaml). Replaced .yaml with .md template.

File: .github/pull-request.md [hunk 1]
```diff
@@ -0,0 +1,5 @@
+Closes [#link issue]
```
Comment (AdrianoKF) [inline .github/pull-request.md:1]: I'd put this at the bottom, so the change summary is the first thing you see when looking at a pull request.

File: .github/pull-request.md [hunk 2]
```diff
@@ -0,0 +1,5 @@
...
+
+**Describe the change**    
+(a clear and concise description of what you have changed)  
+[Your description here]
```
Comment (AdrianoKF) [inline .github/pull-request.md:5]: This is invalid markdown, replace with:
```suggestion
_Your description here_
```

**Comment by AdrianoKF** [comment]
@Maciej818, feel free to merge after the final touches!

**Pull request merged**